doc/man1/req.pod

```diff
@@ -591,6 +601,14 @@ Sample configuration containing all field values:
  [ req_attributes ]
  challengePassword              = A challenge password
 
+Example of giving the most common attributes (subject and extensions)
+on the command line:
+
+ openssl req -new -subj "/C=GB/CN=foo" \
+                  -extension "subjectAltName = DNS:foo.co.uk" \
```

Comment:
On *side* note there is .bar top-domain now, so one can as well use foo.bar as example ;-)

Reply:
Heh...  I'll keep it, though, it's more in line with /C=GB ;-)